### PR TITLE
bringing near-term gas power capacities in the model more in line with historical capacities

### PR DESCRIPTION
**Pull request by cchrisgong**
# Purpose of this PR
- problem: gas power capacity is not in line with historical data (esp. China)
- filed under: https://github.com/remindmodel/development_issues/issues/127 and issue https://github.com/remindmodel/development_issues/issues/177
- fix: 
1) before only 2010 is hard fixed, 2015 is soft fixed; now it is moved to 5 years later, i.e. 2010-2015 are hard fixed, 2020 capacities are soft fixed.  for all regions 
2) getting the newer Chinese gas data (according to Oxford studies), since the Chinese demand is a large share of the global demand, getting the near-term right in China is important. Later this can be incorporated into the pre-processing data input framework
- cs2 of before vs. after: /p/tmp/chengong/gas_fix_china/remind/compScen-compare_gas-fix-2023-01-24_11.13.47-H12

image

## Type of change

(Make sure to delete from the Type-of-change list the items not relevant to your PR)

- [x] Bug fix 
- [ ] Refactoring
- [ ] New feature 
- [X] Minor change (default scenarios show only small differences)
- [ ] Fundamental change
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the coding etiquette
- [x] I have performed a self-review of my own code
- [x] Changes are commented, particularly in hard-to-understand areas
- [x] I have updated the in-code documentation
- [x] I have adjusted reporting where it was needed
- [x] All automated model tests pass (`FAIL 0` in the output of `make test`)

## Further information (optional):

* Test runs are here: 
* Comparison of results (what changes by this PR?):

### CODE REVIEW

File: core/bounds.gms [hunk 1]
```diff
@@ -272,10 +272,35 @@ loop(te$(sameas(te,"spv") OR sameas(te,"wind") OR sameas(te,"windoff")),
 
 $ENDIF.WindOff
 
+***CG: fix CHA gas power capacities: 97 GW by September 2020 (Oxford Institute for Energy Studies, Natural gas in China’s power sector: Challenges and the road ahead, >60% are co-generation, rest are peaking)
+*** for 2018-2022, take 90GW, 90GW*0.6=54, the rest is split between ngcc and ngt
```
Comment (LaviniaBaumstark) [inline core/bounds.gms:276]: please, don't use regioncodes in REMIND GAMS code. Make a new parameter for that or use an existing one and get the info into REMIND via input data. Alternatively, you can introduce a general rule which holds for all regions.

**Review by LaviniaBaumstark — CHANGES_REQUESTED** [review]

**Review by robertpietzcker — CHANGES_REQUESTED** [review]
hi Chris,
Thanks for this improvement of CHA values!

however, the change makes EU results worse - and EU has a larger share gas electricity than CHA historically. So I would like to ask you to remove again the 2010 fix:
vm_cap.lo("2010",regi,te,"1")$pm_histCap("2010",regi,te) = pm_histCap("2010",regi,te);

This should not be needed for getting reasonable 2015/2020 values for CHA.

**Comment by robertpietzcker** [comment]
and (more a general comment, that I should also raise in the REMIND meeting): I think it would be great if you could put the runs into the /p/projects/remind folder and here add the path - if ever questions about the changes come up :-)

File: core/bounds.gms (hunk 1 above)
Comment (cchrisgong) [reply core/bounds.gms]: Thanks, I merged this into mrremind now

**Comment by cchrisgong** [comment]
@robertpietzcker Thanks a lot for the comments! I've removed the 2010 bound and baseline ssp2 converged (I didn't check EU though), and I want to test it for other ssps and for policy runs (waiting for cluster to be back on)

**Comment by cchrisgong** [comment]
@robertpietzcker Thanks a lot for the comments! I've removed the 2010 bound and baseline ssp2 converged (I didn't check EU though), and I want to test it for other ssps and for policy runs (waiting for cluster to be back on)

**Comment by cchrisgong** [comment]
@robertpietzcker 
This is the new cs2 for SE|Gas

China doesn't match that well with 
vm_cap.lo("2020",regi,te,"1")$pm_histCap("2020",regi,te) = 0.75 * pm_histCap("2020",regi,te);
image

Maybe we can increase the 75% to higher? I can try 100% first and see what happens

**Comment by robertpietzcker** [comment]
As said in the messages to Steve before, please use 95%. Never ever use 100%.

**Review by robertpietzcker — APPROVED** [review]

**Comment by robertpietzcker** [comment]
Apart from that: thanks for implementing this!

**Comment by robertpietzcker** [comment]
oh, and could you briefly check if in the regions with too high SE|Elec|Gas (MEA, US, OAS). the 2015/2020 bounds are binding?
Please also change the lower bound for 2015 to 0.95 * historic value, instead of the current 1 * historic value

**Comment by cchrisgong** [comment]
> oh, and could you briefly check if in the regions with too high SE|Elec|Gas (MEA, US, OAS). the 2015/2020 bounds are binding? Please also change the lower bound for 2015 to 0.95 * historic value, instead of the current 1 * historic value

Hey, thanks for the feedback! I checked and the lower bounds are not binding for the three regions 

I'll try 95% for 2015 and 97% for 2020 in the new run

**Comment by cchrisgong** [comment]
image

final comparison

**Comment by cchrisgong** [comment]
the path is 
/p/tmp/chengong/gas_fix_china/remind/SSP2EU-AMT-Base_before_2023-03-15_10.39.00 and SSP2EU-AMT-Base_after_2023-03-15_15.37.40

@robertpietzcker should I still copy them to p/projects? (I think they are on the new cluster? I have to ask Lavinia what to do..)

**Pull request merged**